core/primer.md

```diff
@@ -674,3 +674,64 @@ in the entity in order to be model compliant.
 
 Likewise, implementations of the server must validate the entire entity
 against the new model, not just a subset of the entity's attributes.
+
+# Why are `ancestor` attributes that point to themselves allowed?
+
+The [`ancestor` attribute](spec.md#ancestor-attribute) is used to build a
+hierarchy of Versions to facilitate compatibility checking when the
+[`compatibility` attribute](spec.md#compatibility-attribute) is set.
+Generally speaking, implementations are not expected to allow users to
```

Comment:
This comes really close to saying people SHOULD NOT have multiple roots, and yet we default singleversionroot to 'false'. I'd prefer if we remove this "editorial" comment since it feels inconsistent and it kind of bugs me that we're suggesting people not use it. I don't think we lose anything by dropping it (and the next sentence).